.github/workflows/crafting.yaml

```diff
@@ -18,6 +18,12 @@ jobs:
         run: |
           go mod tidy
           export CGO_ENABLED=0
+          VERSION_FILE="./pkg/cmd/kind/version/version.go"
+          VERSION_CORE="${{ github.ref_name }}"
```

Comment:
`versionCore` in `version.go` doesn't have prefixing `v`, `github.ref_name` if using tag name, it has prefixing `v`